core/src/main/java/de/chkal/mvctoolbox/core/translation/DefaultTranslationResolverFactory.java

```diff
@@ -0,0 +1,33 @@
+package de.chkal.mvctoolbox.core.translation;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.mvc.MvcContext;
+
+/**
+ * Factory for initializing the default translation resolver for the {@link de.chkal.mvctoolbox.core.Toolbox}.
+ *
+ * @author Tobias Erdle
+ * @see de.chkal.mvctoolbox.core.Toolbox
+ */
+@ApplicationScoped
+public class DefaultTranslationResolverFactory {
+
+  /**
+   * Produce the default {@link TranslationResolver} for the {@link de.chkal.mvctoolbox.core.Toolbox}. This will be
+   * an @{@link ApplicationScoped} {@link PropertiesTranslationResolver} which points at a resource bundle named 'messages'
+   * in the projects resources root. This will match for the most applications.
+   * <p>
+   * In case an application needs a more advanced setup or custom {@link TranslationResolver}s, an additional factory with
+   * producer methods can be created.
+   *
+   * @param mvcContext the {@link MvcContext} of the current request
+   * @return the default {@link PropertiesTranslationResolver} for the {@link de.chkal.mvctoolbox.core.Toolbox} bean
+   */
+  @Produces
+  @DefaultTranslationResolver
+  @ApplicationScoped
+  public PropertiesTranslationResolver getDefaultTranslationResolver(final MvcContext mvcContext) {
```

Comment:
I'm not 100% sure, but AFAIK if we replace the return type with `TranslationResolver`, CDI will use the interface to create proxies and therefore we can remove the public default constructor of `PropertiesTranslationResolver`.

Reply:
Ah good catch, I think that was my problem the whole time.

Reply:
Done